cob_hardware_config/cob4-3/config/diagnostics_analyzers.yaml

```diff
@@ -0,0 +1,71 @@
+pub_rate: 1.0 # Optional
+base_path: '' # Optional, prepended to all diagnostic output
+analyzers:
+  computers:
+    type: diagnostic_aggregator/AnalyzerGroup
+    path: Computers
+    analyzers:
+      b1:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: b1
+        contains: 'b1'
+
+#  network:
+#    type: diagnostic_aggregator/AnalyzerGroup
+#    path: Network
+#    analyzers:
+#      wifi:
+#        type: diagnostic_aggregator/GenericAnalyzer
+#        path: wifi
+#        num_items: 1
+#        expected: ['Wifi Status (ddwrt)']
+
+  io:
+    type: diagnostic_aggregator/AnalyzerGroup
+    path: IO
+    analyzers:
+      joystick:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: Joystick
+        contains: 'joy'
+
+  sensors:
+    type: diagnostic_aggregator/AnalyzerGroup
+    path: Sensors
+    analyzers:
+      laser_top:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: Scan Front
+        contains: 'laser_front'
+      laser_rear:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: Scan Right
+        contains: 'laser_right'
+      laser_front:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: Scan Left
+        contains: 'laser_left'
+
+  motors:
+    type: diagnostic_aggregator/AnalyzerGroup
+    path: Actuators
+    analyzers:
+      base:
+        type: diagnostic_aggregator/GenericAnalyzer
+        path: Base
+        startswith: '/base'
+
+  safety:
+    type: diagnostic_aggregator/GenericAnalyzer
+    path: Safety
+    contains: 'safety'
+
+## remove diagnostics from gazebo in simulated case
+  simulation:
+    type: diagnostic_aggregator/DiscardAnalyzer
+    path: none
+    contains: 'joint'
+  simulation2:
+    type: diagnostic_aggregator/DiscardAnalyzer
+    path: none
+    contains: 'Controller'
```

Comment:
I will open a new pull request removing these lines for all the robots, as soon as this PR is merged.